securesignals-gma-dev-app/src/main/java/com/uid2/dev/utils/ContextEx.kt

```diff
@@ -0,0 +1,10 @@
+package com.uid2.dev.utils
+
+import android.content.Context
+import android.content.pm.PackageManager
+import android.os.Bundle
+
+fun Context.getMetadata(): Bundle = packageManager.getApplicationInfoCompat(
```

Comment:
We don't want to expose this function publicly

```suggestion
internal fun Context.getMetadata(): Bundle = packageManager.getApplicationInfoCompat(
```